server/config/routes.js

```diff
@@ -261,6 +261,13 @@ exports.configure = function configure(app) {
   /*  Inventory and Stock Management */
   app.post('/inventory/metadata', inventory.createInventoryItems);
   app.get('/inventory/metadata', inventory.getInventoryItems);
+  app.get('/inventory/metadata/:uuid', inventory.getInventoryItemsById);
+  app.put('/inventory/metadata/:uuid', inventory.updateInventoryItems);
+
+  /**
+   * @fixme: these routes below are deprecated
```

Comment:
You can also use the `@deprecated` tag to show that things are deprecated.  Most text editors will highlight it.